eunkyung/app.js

```diff
@@ -34,25 +34,141 @@ app.get('/ping', (req, res) => {
 })
 
 app.post('/sign_up', async (req, res, next) => {
-    const { name, password, email } = req.body
+    const { name, password, email, profile_image } = req.body
     await mysqlDataSource.query(
         `INSERT INTO users (
             name, 
             password, 
-            email
+            email,
+            profile_image
             ) 
             VALUES (
+                ?,
                 ?,
                 ?,
                 ?
                 );`,
-        [name, password, email]
+        [name, password, email, profile_image]
     )
     res.status(201).json({ message: 'userCreated' })
 })
 
 
 
+app.post('/posting', async (req, res) => {
+    const { title, content, post_image_url, user_id } = req.body
```

Comment:
camelCase 수정이 필요한 부분 모두 수정해주세요!

```suggestion
    const { title, content, postImageUrl, userId } = req.body
```